core/modules/comment/views/views_handler_filter_comment_user_uid.inc

```diff
@@ -10,6 +10,7 @@
  *
  * @ingroup views_filter_handlers
  */
+#[\AllowDynamicProperties]
 class views_handler_filter_comment_user_uid extends views_handler_filter_user_name {
```

Comment:
I think, the base class for that class is views_handler_filter (modules/views/handlers/views_handler_filter.inc).